src/ParallelAlgorithms/ApparentHorizonFinder/InterpolateVolumeVars.hpp

```diff
@@ -0,0 +1,30 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <unordered_map>
+
+#include "DataStructures/LinkedMessageId.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "ParallelAlgorithms/ApparentHorizonFinder/Storage.hpp"
+
+template <size_t VolumeDim>
+class Domain;
+template <size_t VolumeDim>
+class ElementId;
+
+namespace ah {
+/*!
+ * \brief Interpolate volume data from any new elements to the target points
```

Comment:
Expand on this documentation a bit.  What are "new elements"?  None of the const arguments contain any data, so what are the inputs and outputs?